app/components/in-view.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
```

Comment:
I checked what actually takes longer time and it turns out, that searching on Mopidy and not inserting result into the DOM, so we probably don't need this infinite scroll optimization at all.